### DISCUSSION TITLE
Redlinning 41 zonal instructions?

### DISCUSSION THREAD

**Discussion by VictoriAragon** [earthlab-education/Earth-Analytics-AY24#625] [Q&A]
Did I miss this being mentioned where it would be available? I can't seem to find the video or text instructions in the  textbook.

**Comment by eculler** [comment]
Hi @VictoriAragon, I sent the video for 41 and 42 out on Canvas because I'm having an issue with the website rendering. Here it is if you didn't get that announcement: https://www.canva.com/design/DAGWGOJ1gvo/qNtsBehK4sXPGcp8sP7fxw/watch?utm_content=DAGWGOJ1gvo&utm_campaign=designshare&utm_medium=link&utm_source=editor

Videos for the final project are also going out on Canvas at the moment.

**Reply by VictoriAragon** [reply]
Thank you Elsa. I appreciate the fast reply.
